### PR TITLE
updated Symfony and PHP

### PR DESCRIPTION
**Pull request by borstl**
- added support for Symfony 4.4, 5.4 and 6.0
- added support for PHP >=7.4
- added psalm
- removed support for Symfony 3.4 and PHP <7.4

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 /vendor/
 composer.lock
+/.phpunit.result.cache
+/.idea
```
Comment (loevgaard) [inline .gitignore]: I don't want this in .gitignore. You should have it in your own global gitignore :)

**Comment by loevgaard** [comment]
Thanks for the PR, @borstl! Let's just fix the one comment I have and I will merge and release :)

**Comment by loevgaard** [comment]
Released: https://github.com/Setono/TinyPngBundle/releases/tag/v1.1.0 :)

File: .gitignore (hunk 1 above)
Comment (borstl) [reply .gitignore]: Understandable. Thank you for the quick response :)

**Pull request merged**